CMakeLists.txt

```diff
@@ -351,7 +361,7 @@ macro(commom_lib_settings lib write )
         )
         target_include_directories(
             ${lib}
-        PUBLIC
+        PRIVATE
             $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib/dfdutils>
             $<INSTALL_INTERFACE:lib/dfdutils>
```

Comment:
Is this line really necessary? `lib/dfdutil` is never installed.

Reply:
This is analogue to the transcoder headers